README.md

```diff
@@ -4,16 +4,107 @@
 [![Build Status](https://travis-ci.org/jboss-fuse/vscode-datavirt.svg?branch=master)](https://travis-ci.org/jboss-fuse/vscode-datavirt)
 [![License](https://img.shields.io/badge/license-Apache%202-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
+## Prerequisites
+
+- recent Visual Studio Code
+
+
 ## Features
 
-...
+Red Hat Integration - Data Virtualization Tooling enables you to create a Virtual Database definition file (VDB) and to visualize and manipulate its contents inside [VS Code](https://code.visualstudio.com/).
 
+### The Data Virtualization view
+The extension provides its own view to [VS Code](https://code.visualstudio.com/) which is called `Data Virtualization`.
 
-## How to contribute
+![Data Virtualization View](./images/dv_view.png)
 
-See [Contributing guide](./Contributing.md)
+### Creating a Virtual Database definition
 
+There are two ways to create a new VDB file. 
 
-## Prerequisites
+Option 1 would be to open up the command palette (`F1`) of [VS Code](https://code.visualstudio.com/) and to enter `New Virtual Database` into the search field like shown in below screenshot.
```

Comment:
I would somehow highlight "Option 1" and "Option 2" inside text to be more visible that there are two approaches..

maybe just ":" would be enough or add new paragraph...

Reply:
I think the images punctuate the options without additional formatting, but perhaps @MelissaFlinn will have another take on this?